content/yaml-publishing/google-play.md

```diff
@@ -131,24 +131,16 @@ publishing:
 
     In order to publish to a track (e.g. `internal`) and immediately promote the same release to another track (e.g. `alpha`) to reach a wider audience, additionally configure the `release_promotion` section.
 
-4. If your application supports [in-app updates](https://developer.android.com/guide/playcore/in-app-updates), Codemagic allows setting the update priority. Otherwise, `in_app_update_priority` can be omitted or set to `0`.
-
-5. In addition, Codemagic supports [staged releases](https://support.google.com/googleplay/android-developer/answer/6346149?hl=en), allowing users to choose which fraction of the testers or users get access to the application. To release to everyone, omit `rollout_fraction` from codemagic.yaml.
-
-6. The field `changes_not_sent_for_review` is required if you are getting the error _**Changes cannot be sent for review automatically. Please set the query parameter changesNotSentForReview to true. Once committed, the changes in this edit can be sent for review from the Google Play Console UI.**_
+{{<notebox>}}
+**Note:** You can use the "**Wear OS Only**" track to manage Wear OS releases in Play Console. 
+To target "**Wear OS Only**" track, add **wear:** in the track name.
 
-7. If your changes are sent to review automatically, but the field is still set to `true`, you may get the error _**Changes are sent for review automatically. The query parameter changesNotSentForReview must not be set.**_
+`track: wear:internal`
```

Comment:
perhaps add this line above: _To target "**Wear OS Only**" track, add **wear:** in the track name: `track: wear:internal`_